octavia-cli/build.gradle

```diff
@@ -17,6 +17,6 @@ tasks.register('generate').configure {
     dependsOn generateApiClient
 }
 
-tasks.named('installReqs').configure {
-    dependsOn generateApiClient
-}
+//tasks.named('installReqs').configure {
+//    dependsOn generateApiClient
+//}
```

Comment:
This broke.... something cc @postamar

Reply:
You need to leave this in for `octavia-cli` to build.

The real question is: does it need to be part of the gradle build at all? It doesn't seem like a piece of software that's actively used.